src/components/lotes/LoteForm.jsx

```diff
@@ -109,6 +115,15 @@ class LoteForm extends Component {
     }));
   }
 
+  updateTecnica(obras) {
+    const tecnicas = obras.map((obra) => {
```

Comment:
While this approach works I feel it breaks the existing pattern. I feel it would be better for these values to be set to the state and updated whenever the Obras group updates.

Reply:
We don't have time to refactor stuff that works

Reply:
the reason i didnt do it this way is because on `removeObraFromGroup()` we would have to check if tecnica of the obra being removed exists on any other obra in group, which is of course possible but tedious.

Reply:
Yh I know but this is all going to be refactored to include the up/down functionality. But then again that is another branch so doesnt matter.